playground/pages/page1.vue

```diff
@@ -0,0 +1,24 @@
+<script setup lang="ts">
+// for testing purposes with auto-import disabled
+// import { TresCanvas } from '#components'
+// import { onMounted, useRenderLoop } from '#imports'
+
+const { resume } = useRenderLoop()
```

Comment:
this can be moved inside the `onMounted` hook

Reply:
It can be removed since is a leftover trying to fix the issue you solved.